theme_yes/views/snippets/s_masonry_block.xml

```diff
@@ -2,46 +2,26 @@
 <odoo>
 
 <template id="s_masonry_block_default_template" inherit_id="website.s_masonry_block_default_template">
-    <!-- Image Block -->
-    <xpath expr="//div[hasclass('o_grid_item_image')]" position="attributes">
-        <attribute name="class" add="col-lg-4 g-col-lg-4 border rounded" remove="col-lg-6 g-col-lg-6" separator=" "/>
-        <attribute name="style" add="border-color: rgb(255, 255, 255) !important; border-width: 10px !important; border-radius: 14px !important; grid-area: 1 / 1 / 11 / 5;" separator="; "/>
-    </xpath>
-    <!-- Image Block - Image -->
-    <xpath expr="//img" position="attributes">
-        <attribute name="style" add="border-radius: 4px !important;" separator="; "/>
-    </xpath>
-
     <!-- Little Blocks -->
     <!-- Little block #01 - Title -->
     <xpath expr="//h3" position="replace" mode="inner">
         Caterers
     </xpath>
     <!-- Little block #01 - Text -->
-    <xpath expr="//*[hasclass('col-lg-3')]//p" position="replace">
+    <xpath expr="//*[hasclass('col-lg-4')]//p" position="replace">
         <p>Check out our list of favorite caterers.</p>
         <p><a href="#" class="btn btn-secondary rounded-circle">Find a Caterer</a></p>
     </xpath>
-     <!-- Little block #01 - Main div -->
-     <xpath expr="//div[hasclass('col-lg-3')]" position="attributes">
-        <attribute name="class" add="col-lg-4 g-col-lg-4 border rounded" remove="col-lg-3 g-col-lg-3" separator=" "/>
-        <attribute name="style" add="border-color: rgb(255, 255, 255) !important; border-width: 10px !important; border-radius: 14px !important; grid-area: 1 / 5 / 6 / 9;" separator="; "/>
-    </xpath>
 
     <!-- Little block #02 - Title -->
     <xpath expr="(//h3)[2]" position="replace" mode="inner">
         Venues
     </xpath>
     <!-- Little block #02 - Text -->
-    <xpath expr="//*[hasclass('col-lg-3')]//p" position="replace">
+    <xpath expr="//*[hasclass('col-lg-4')][2]//p" position="replace">
         <p>Check out our list of favorite venues.</p>
         <p><a href="#" class="btn btn-secondary">Find a venue</a></p>
```

Comment:
It seems that the texts of "Attire" and "Venues" have been inverted :+1: 
![image](https://github.com/user-attachments/assets/3f35ee79-7307-4cdf-8035-c6addc478ca8)

Surely because the second "h3" is not in the second "col-lg-4" (which is why the order of the elements is important (see my comment about the mobile order in the community PR) :wink: 

By the way, I've always found this image to be a bit of bad quality (like it seems pixelated) => maybe you could replace it (just saying) ? :slightly_smiling_face: